CHANGELOG.md

```diff
@@ -103,19 +103,26 @@
   ```
 * Added `RealmValueType` enum that contains all the possible types that can be wrapped by a `RealmValue`. (PR [#1469](https://github.com/realm/realm-dart/pull/1469))
 * Added support for accessing `Set` and `Map` types using the dynamic object API - `obj.dynamic.getSet/getMap`. (PR [#1533](https://github.com/realm/realm-dart/pull/1533))
-
+* Improve performance of object notifiers with complex schemas and very simple changes to process by as much as 20% ([PR #7424](https://github.com/realm/realm-core/pull/7424)).
+* Improve performance with very large number of notifiers as much as 75% ([PR #7424](https://github.com/realm/realm-core/pull/7424)).
+* Add support to synchronize collections embedded in Mixed properties and other collections (except sets) ([PR #7353](https://github.com/realm/realm-core/pull/7353) Core v14.2.0-12-g95c6efce8).
```

Comment:
We likely want to update to 14.3.0 before we release publicly, so these changelog entries should ideally refer to the tagged version of Core.

Reply:
I was thinking we could update then refs then. I'm a little hesitant to refer to 14.3 when it is not released yet, and 14.2 does not contain the fix.

Reply:
Fair enough, we just need to remember to go through those and fix them.